spmk-musica-ecclesiastica.csl

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="pl-PL">
+  <info>
+    <title>Styl SPMK Musica Ecclesiastica (SPMK Musica Ecclesiastica)</title>
+    <id>http://www.zotero.org/styles/spmk-musica-ecclesiastica</id>
+    <link href="http://www.zotero.org/styles/spmk-musica-ecclesiastica" rel="self"/>
+    <author>
+      <name>Czesław Grajewski</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <summary>Styl cytowań dla rocznika Musica Ecclesiastica wydawanego przez SPMK (Polska).</summary>
+    <updated>2025-10-08T00:00:00Z</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author-citation">
+    <names variable="author">
+      <name form="long" initialize="true" initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="author-bibliography">
+    <names variable="author" delimiter=", ">
+      <name form="long" name-as-sort-order="all" sort-separator=" ">
+        <name-part name="family"/>
+        <name-part name="given" prefix=" "/>
+      </name>
+    </names>
+  </macro>
+  <macro name="title-full">
+    <text variable="title" font-style="italic"/>
+  </macro>
+  <macro name="title-short">
+    <text variable="title-short" font-style="italic"/>
+  </macro>
+  <macro name="journal-info">
+    <group delimiter=" ">
+      <text variable="container-title" prefix="„" suffix="”"/>
+      <text variable="volume"/>
+      <date variable="issued" prefix=" (" suffix=")">
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="pages-citation">
+    <text variable="locator" prefix="s. "/>
```

Comment:
You have a variety of hardcoded terms/labels. Replace this with the locator label (in a group with a space as a delimiter)